ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/additional-ca-always/provision/cucushift-installer-rehearse-aws-ipi-additional-ca-always-provision-chain.yaml

```diff
@@ -0,0 +1,15 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-additional-ca-always-provision
```

Comment:
How about remove `always` here and set `ADDITIONAL_TRUST_BUNDLE_POLICY` on job config level, so that we can easily add other type of testing if needed.

Reply:
@jianlinliu I renamed the related `additional-ca-always` workflow and chain to `additional-ca-policy`, and removed the `ADDITIONAL_TRUST_BUNDLE_POLICY: Always` from the provision chain settings. 

But actually I think we can't use this chain and workflow for other testing usage, since there're only two available options for "ADDITIONAL_TRUST_BUNDLE_POLICY", and "Proxyonly" is the default one that's being tested in all the other jobs. The only scenario when we want to set ADDITIONAL_TRUST_BUNDLE_POLICY in install-config is setting it to "Always".

Reply:
> emoved the `ADDITIONAL_TRUST_BUNDLE_POLICY: Always` from the provision chain settings.

This makes provision chain more flexible.